### PR TITLE
refactoring maintenance section

### PR DESCRIPTION
**Pull request by atarkowska**
template adjustment for https://github.com/openmicroscopy/openmicroscopy/pull/3326

### CODE REVIEW

File: etc/templates/nginx.conf.system.template [hunk 1]
```diff
@@ -2,16 +2,18 @@
         listen       %(HTTPPORT)d;
         server_name  $hostname;
 
+        error_page 502  /500.html;
```
Comment (manics) [inline etc/templates/nginx.conf.system.template]: This sets the error_page for `/`, but if a prefix has been specified shouldn't the error handler only apply to e.g. `/omero`?

Comment (atarkowska) [inline etc/templates/nginx.conf.system.template]: I will move error_page into location seems to be possible

**Comment by atarkowska** [comment]
@manics is that better now?

**Comment by atarkowska** [comment]
with that trick admin always sees maintenance page if web is turned off, admin doesn't need to create custom pages, etc.

**Comment by manics** [comment]
Medged, thanks @aleksandra-tarkowska

**Pull request merged**